_posts/2024-12-05-this-month-in-servo.md

```diff
@@ -1,40 +1,52 @@
 ---
 layout:     post
 tags:       blog
-title:      "This month in Servo: and more!"
+title:      "This month in Servo: grid layout, parallel flexbox, smaller binaries, and more!"
 date:       2024-12-05
 summary:    ao!! wrrrrao!!
 categories:
 ---
 
+Servo now supports **CSS grid layout** (@nicoburns, @taniishkaa, #32619, #34352, #34421), **:is() and :where() selectors** (@mrobinson, #34066), and the **getClientRects()** method on Element (@chickenleaf, #34025)!
```

Comment:
getClientRects already existed. We just changed the return type to match some specification updates.

Reply:
fixed in c2e66f694653e490261e440ec2a24c249bf17b3c

Reply:
fixed in efc40787fa225cafa8e32fd09b621dd460be0ca7